tpm/tpm_test.go

```diff
@@ -232,6 +232,33 @@ func TestResizeableSlice(t *testing.T) {
 	}
 }
 
+func TestNVDefineSpace(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+	index := tpmutil.Handle(0x40000001)
+	permissions := []uint32{nvPerWriteSTClear}
+	length := uint32(54)
+	ownerAuth := getAuth(ownerAuthEnvVar)
+	_, err := NVDefineSpace(rwc, index, length, permissions, ownerAuth)
+	if err != nil {
+		t.Fatal("Couldn't define space:", err)
+	}
+
+}
+
+func TestNVWriteValue(t *testing.T) {
+	rwc := openTPMOrSkip(t)
+	defer rwc.Close()
+	index := uint32(0x40000001)
+	ownerAuth := getAuth(ownerAuthEnvVar)
+	randomdata := make([]byte, 20)
+	rand.Read(randomdata)
+	_, err := NVWriteValue(rwc, index, 0, randomdata, ownerAuth)
+	if err != nil {
```

Comment:
Same comment as in the previous test: if it's not possible to test anything to do with the return value, then fold the previous line into this one. However, I'd prefer to check the return value if possible.